src/App.tsx

```diff
@@ -1,7 +1,29 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {};
+
+type State = {
+  x: string;
+};
+
+export class App extends React.Component<Props, State> {
+  state = {
+    x: 'key',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ x: event.key });
+    });
+  }
+
+  render(): React.ReactNode {
+    return (
+      <div className="App">
+        <p className="App__message">
+          {`The last pressed key is [${this.state.x}]`}
```

Comment:
But what if nothing was pressed? You should return another message in that case
![image](https://user-images.githubusercontent.com/96409563/226717804-245b7242-0258-4809-aaf3-f0f762a61697.png)